packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -1038,6 +1044,7 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 	}
 
 	public async getPendingBlobs(waitBlobsToAttach?: boolean): Promise<IPendingBlobs> {
+		this.blockUploading = true;
 		if (waitBlobsToAttach) {
 			await this.shutdownPendingBlobs();
```

Comment:
can we reduce concepts. here we have block, wait, and shutdown, and they are all for basically the same thing, i'd rather them use the same terms.